### DISCUSSION TITLE
Scrolling is capped to 60Hz on high refresh rate

### DISCUSSION THREAD

**Discussion by zoeleu** [ppy/osu#24148] [Q&A]
### Type

Performance

### Bug description

Scrolling on a high refresh rate screen causes the scroll to be on 60fps, even if main gameplay is uncapped.

### Version

2023.621.0-lazer

### Logs

[updater.log](https://github.com/ppy/osu/files/11972988/updater.log)
[database.log](https://github.com/ppy/osu/files/11972990/database.log)
[input.log](https://github.com/ppy/osu/files/11972991/input.log)
[network.log](https://github.com/ppy/osu/files/11972993/network.log)
[performance.log](https://github.com/ppy/osu/files/11972994/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11972995/runtime.log)

**Comment by peppy** [comment]
This is impossible.

Please provide a video with frame graphs visible (ctrl+F11 twice) if you want further support.

**Comment by zoeleu** [comment]
The performance drops to around 60 fps when scrolling, particularly through the modifiers menu.
https://github.com/ppy/osu/assets/69425093/370d2c45-8fb5-4c22-be7d-70238954259a

**Reply by integerrr** [reply]
Need frame graphs, Ctrl+F11 again please
